src/openms/source/FORMAT/MascotRemoteQuery.cpp

```diff
@@ -171,6 +171,7 @@ namespace OpenMS
     QNetworkRequest request(url);
 
     QString boundary = boundary_.toQString();
+    boundary = boundary.toUtf8();
```

Comment:
this is a no-op?!
on the right side, you create a qbytearray, which you assign to the QString again (which has an assignment operator for it).

better:
```
QByteArray boundary = boundary_.toQString().toUtf8;
```
and then you can also omit the `.toUtf8()` call below.

Reply:
I am assuming you meant
```
**QString** boundary = boundary_toQString().toUtf8()
```

Reply:
no. I meant `QByteArray boundary = boundary_.toQString().toUtf8;`, because `QString boundary = boundary_.toQString().toUtf8()` is again a no-op.

Reply:
but is this `toUtf8` a thing or you meant `toUtf8()`